orcid-scheduler-web/src/main/java/org/orcid/scheduler/indexer/impl/OrcidRecordIndexerImpl.java

```diff
@@ -113,10 +113,10 @@ public void reindexRecordsOnSolr() {
     @Override
     public void reindexV3RecordsOnS3() {
         this.processProfilesWithFlagAndAddToMessageQueue(IndexingStatus.S3_V3_REINDEX);
-    }
+    }    
     
-    @Override
-    public void processProfilesWithForceIndexingFlagAndAddToMessageQueue() {
+    @Override 
+    public void processProfilesWithFroceIndexingFlagAndAddToMessageQueue() {
```

Comment:
There is a typo in the method name it says `..FroceIndexing...` and it should be `...ForceIndexing...`